From fc9bae7de600c286aa2f6a6c4db5aa2b9fd06845 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Wed, 2 Apr 2014 10:19:44 +0800
Subject: [PATCH 0373/1074] ENGR00300157: ARM: dts: imx6sl-evk-common: correct
 the pad setting of EPDC_PWRCTRL3

align with the same setting with v3.0.35, otherwise, cause 'POR' reset
if EPDC daughter board attached sometimes.

Signed-off-by: Robin Gong <b38343@freescale.com>
[Original patch taken from git://git.freescale.com/imx/linux-2.6-imx.git]
Signed-off-by: Biyao Zhai <biyao.zhai@windriver.com>
---
 arch/arm/boot/dts/imx6sl-evk-common.dtsi |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sl-evk-common.dtsi b/arch/arm/boot/dts/imx6sl-evk-common.dtsi
index a1a3969..0eb4295 100644
--- a/arch/arm/boot/dts/imx6sl-evk-common.dtsi
+++ b/arch/arm/boot/dts/imx6sl-evk-common.dtsi
@@ -350,7 +350,7 @@
 				MX6SL_PAD_EPDC_PWRCTRL0__GPIO2_IO07 0x80000000
 				MX6SL_PAD_EPDC_PWRWAKEUP__GPIO2_IO14 0x80000000
 				MX6SL_PAD_EPDC_PWRINT__GPIO2_IO12 0x80000000
-				MX6SL_PAD_EPDC_PWRCTRL3__GPIO2_IO10 0x170b0
+				MX6SL_PAD_EPDC_PWRCTRL3__GPIO2_IO10 0x17000
 				MX6SL_PAD_EPDC_PWRCTRL2__GPIO2_IO09 0x80000000
 				MX6SL_PAD_KEY_COL6__GPIO4_IO04    0x110b0
 				MX6SL_PAD_ECSPI2_MISO__GPIO4_IO14 0x17000
-- 
1.7.5.4

